test/test_jit.py

```diff
@@ -12512,6 +12512,18 @@ def is_tensor_value(item):
                     continue
                 self.assertEqual(item[1], loaded_item)
 
+    def test_script_recurse(self):
+        def a_python_fn(a, b, c):
+            return a + b + c
+
+        @torch.jit._recursive_script
```

Comment:
If we are putting this in experimentally, can we do it as a thread local flag, like our other experiments? Otherwise it ends up being threaded through a huge amount of code which is a recipe for maintenance problems and merge conflict.